pkg/clients/clients.go

```diff
@@ -53,6 +53,12 @@ type Client struct {
 	Permissions []string    `json:"permissions,omitempty"`
 }
 
+type ConnectionStatus struct {
+	ChannelId string `json:"channel_id"`
+	ThingId   string `json:"thing_id"`
+	Status    string `json:"status"`
+}
```

Comment:
We can have like new `int` type for `ConnectionStatus` and with const values as `iota` and function to convert `int` type of `ConnectionStatus` to string.